src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/config/studio/workflow/notification-config.xml

```diff
@@ -22,6 +22,7 @@
 
 -->
 <notificationConfig>
+    <version>2</version>
```

Comment:
Please add the version for the other blueprints too, and remember to send a PR for video-center too